wordpress-mobile/WordPress-Android
# Issue/17260 wordpress module asynctask warning commentdetailactivity

Fixes [#17260](https://github.com/wordpress-mobile/WordPress-Android/issues/17260)

To test:

1. Open App and Login
2. Go to `My Site` tab
3. Select `Comments` in home
4. In `Comments` page there is list of comments
5. Open anyone of the comment
6. Swipe left and right to go previous and next comment
7. Replay or Like or marked as spam the comment
8. Check it updating



## Regression Notes
1. Potential unintended areas of impact
`Comments` screen and it's functionality

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually test this page. Working good.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


UI Changes testing checklist:
